test/servers.ts

```diff
@@ -0,0 +1,5 @@
+export const servers = {
```

Comment:
Remember, this file should never be checked in. Let's add the path to `.gitignore` and then delete this file.